app/src/main/res/layout/fragment_driving_license.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".presentation.authentification.register.verification.DrivingLicenseFragment"
+    tools:ignore="ContentDescription">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/authentication_title_margin_top"
+        android:orientation="horizontal"
+        tools:ignore="UseCompoundDrawables">
+
+        <ImageView
+            android:id="@+id/backIV"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/horizontal_margin"
+            android:layout_marginTop="@dimen/driving_license_back_arrow_margin_top"
+            android:src="@drawable/ic_driving_license_arrow_back" />
+
+        <TextView
+            android:id="@+id/titleTV"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/horizontal_margin"
+            android:fontFamily="@font/bai_jamjuree_semi_bold"
+            android:gravity="center"
+            android:text="@string/authentication_driving_license_title"
+            android:textColor="@color/secondary_text_color"
+            android:textSize="@dimen/heading3_text_size" />
+    </LinearLayout>
+
+    <ImageView
+        android:id="@+id/drivingLicenseIV"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/driving_license_margin_top"
+        android:scaleType="centerCrop"
+        android:src="@drawable/ic_driving_license" />
+
+    <TextView
+        android:id="@+id/headingTV"
+        style="@style/Heading1"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/horizontal_margin"
+        android:layout_marginTop="@dimen/driving_license_heading_margin_top"
+        android:layout_marginEnd="@dimen/horizontal_margin"
+        android:text="@string/authentication_driving_license_heading"
+        android:textColor="@color/secondary_text_color" />
+
+    <TextView
+        android:id="@+id/contentTV"
+        style="@style/TextAppearance"
```

Comment:
Nice use of the styles!

But try to be a bit more explicit: `TextAppearance` does not say anything. 
`headingTv` properly uses this concept: you named it `Heading1`, which is way more explicit.